torch/_dynamo/utils.py

```diff
@@ -1454,7 +1454,11 @@ def to_tensor(t):
                 # false alarms. We use multiplier of 3 instead of 2 to avoid these false alarms.
                 multiplier = 3.0 if res.dtype == torch.bfloat16 else 2.0
 
-                if (
+                if fp64_ref.numel() <= 10 and tol >= 4 * 1e-2:
+                    multiplier = 8.0
+                elif fp64_ref.numel() <= 500 and tol >= 4 * 1e-2:
+                    multiplier = 5.0
+                elif (
```

Comment:
Hmm, i'm not sure we should do this. i guess the logic is that if it's a smaller tensor it must have come via reduction of more elements so will accumulate error. 

Still, think it would be better to just opt in for these models. happy to hear other thoughts.

Reply:
My understanding here is, for independent and identically distributed random variables, the more number we average together (like what we do for RMSE here), the smaller the variance (noise).

So a smaller tensor in general has bigger noise for RMSE.

I think it's a good point to make this opt in